lib/paypal_nvp.rb

```diff
@@ -58,6 +69,9 @@ def call_paypal(data)
       http.verify_mode = OpenSSL::SSL::VERIFY_NONE
     end
 
+    http.open_timeout = @open_timeout
+    http.read_timeout = @read_timeout
```

Comment:
I think setting them to the defaults in `Net::HTTP` when `@open_timeout` or `@read_timeout` is nil is probably the most expected behaviour?